Datadog/Datadog.xcodeproj/project.pbxproj

```diff
@@ -552,6 +552,8 @@
 		9E68FB55244707FD0013A8AA /* ObjcExceptionHandler.m in Sources */ = {isa = PBXBuildFile; fileRef = 9E68FB53244707FD0013A8AA /* ObjcExceptionHandler.m */; };
 		9E68FB56244707FD0013A8AA /* ObjcExceptionHandler.h in Headers */ = {isa = PBXBuildFile; fileRef = 9E68FB54244707FD0013A8AA /* ObjcExceptionHandler.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		9EE5AD8226205B82001E699E /* DDNSURLSessionDelegateTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9EE5AD8126205B82001E699E /* DDNSURLSessionDelegateTests.swift */; };
+		A703694D2BB2E30100C66C36 /* DataStoreMock.swift in Sources */ = {isa = PBXBuildFile; fileRef = A703694C2BB2E30100C66C36 /* DataStoreMock.swift */; };
+		A703694E2BB2E30100C66C36 /* DataStoreMock.swift in Sources */ = {isa = PBXBuildFile; fileRef = A703694C2BB2E30100C66C36 /* DataStoreMock.swift */; };
```

Comment:
question/ This seems wrong, we're not adding any new files in this PR. The `DataStoreMock` is perhaps from before-rebase (I added this mock through separate branch).